blog/force-app/main/default/aura/markdownBlogging/markdownBlogging.cmp

```diff
@@ -0,0 +1,55 @@
+<aura:component implements = "force:appHostable,flexipage:availableForAllPageTypes,flexipage:availableForRecordHome,force:hasRecordId,forceCommunity:availableForAllPageTypes,force:lightningQuickAction" access = "global" >
```

Comment:
these are the elements you really need here  --> force:appHostable,flexipage:availableForRecordHome,force:hasRecordId